spec/searchable_by_spec.rb

```diff
@@ -22,6 +22,14 @@
     expect(sql).not_to include(%("posts"."id"))
     expect(sql).to include(%("posts"."title" LIKE 'foo\\%bar%'))
     expect(sql).to include(%("posts"."body" LIKE '%foo\\%bar%'))
+
+    sql = User.search_by('Uni*dom').to_sql
+    expect(sql).to include(%("users"."country" LIKE '%Uni%dom%'))
+
+    sql = User.search_by('Uni * dom').to_sql
```

Comment:
I don't like this, if s/o wanted to do that, they would need to use `User.search_by('"Uni * dom"').to_sql`